### PR TITLE
Replace Files with BufferedReader

### LINKED ISSUE #108: FileSystemNotFoundException when running from jar
Reported from: https://stackoverflow.com/questions/62611044/filesystemnotfoundexception-while-running-a-javafx-app-on-terminal

After this [refactoring](https://github.com/FXyz/FXyz/commit/a8e519acc06328d8d1f55675df8cb9f8ff94cfc3#diff-a5ab33f08c3367762325fe178b0af5c3), when an obj model is bundled within a jar and `Importer3D.loadAsPoly(getClass().getResource(".."))` is called, there is this exception:

```
Exception in thread "JavaFX Application Thread" java.nio.file.FileSystemNotFoundException
    at jdk.zipfs/jdk.nio.zipfs.ZipFileSystemProvider.getFileSystem(ZipFileSystemProvider.java:172)
    at jdk.zipfs/jdk.nio.zipfs.ZipFileSystemProvider.getPath(ZipFileSystemProvider.java:158)
    at java.base/java.nio.file.Path.of(Path.java:208)
    at java.base/java.nio.file.Paths.get(Paths.java:98)
    at org.fxyz3d.importers.obj.ObjImporter.read(ObjImporter.java:115)
    at org.fxyz3d.importers.obj.ObjImporter.loadAsPoly(ObjImporter.java:102)
```

Similar issue happens using jlink, up until Java 13.

Proposal: Add a FileSystem if it doesn't exist.

### PR DESCRIPTION
**Pull request by nlisker**
Fixes #108

### CODE REVIEW

**Comment by nlisker** [comment]
@jperedadnr Can I integrate this or do you want to review?

File: FXyz-Importers/src/main/java/org/fxyz3d/importers/obj/ObjImporter.java [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2019 F(X)yz
+ * Copyright (c) 2020 F(X)yz
```
Comment (jperedadnr) [inline FXyz-Importers/src/main/java/org/fxyz3d/importers/obj/ObjImporter.java:2]: Modify to: ' 2019, 2020,`

Comment (nlisker) [reply FXyz-Importers/src/main/java/org/fxyz3d/importers/obj/ObjImporter.java]: Updated. Anything else?

Comment (jperedadnr) [reply FXyz-Importers/src/main/java/org/fxyz3d/importers/obj/ObjImporter.java]: Thx, you can merge now :)

**Pull request merged**